coding/src/lib.rs

```diff
@@ -10,4 +10,306 @@
     html_favicon_url = "https://commonware.xyz/favicon.ico"
 )]
 
+use bytes::Buf;
+use commonware_codec::Codec;
+use rand_core::CryptoRngCore;
+
 pub mod reed_solomon;
+pub use reed_solomon::ReedSolomon;
+
+/// Configuration common to all encoding schemes.
+pub struct Config {
```

Comment:
Can we implement `Codec` on this type? I _think_ I'll need to share it around as a part of #1520, since [`Supervisor::participants`](https://docs.rs/commonware-consensus/latest/commonware_consensus/trait.Supervisor.html#tymethod.participants) can fluctuate between views.